### DISCUSSION TITLE
Translating dataset.json in 5.1

### DISCUSSION THREAD

**Discussion by franklinbenini** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#92] [Q&A] [answered]
Release 5.1 introduced "Character Generation". But how can I localize it ? Files on "lang" folder don't have strings for dataset.json :(

**Comment by aMediocreDad** [comment, marked as answer]
Hey, the best way to do it would be to translate the dataset.json and distribute it in a module. 

Of course, we can also host translations of the json file in the system itself. Just open a pull request with the json file.

There is unfortunately no way to do full localization in Foundry without relying on thirdparty modules.
